src/components/Dropdown/Dropdown.js

```diff
@@ -50,6 +50,10 @@ export default class Dropdown extends PureComponent {
     }
   }
 
+  componentDidMount() {
```

Comment:
Could we add the accompanying `componentWillUnmount` to remove the listener?

Reply:
yes,  I will add this